test/N3Store-test.js

```diff
@@ -1036,6 +1037,38 @@ describe('N3Store', function () {
       shouldIncludeAll(store.getQuads(null, null, null, 'null'), ['null', 'null', 'null', 'null'])();
     });
   });
+
+  describe('N3Store Source interface', function () {
+    var store = new N3Store();
+
+    it('should create a Stream', function () {
+      var stream = store.match();
+      (stream instanceof N3Stream).should.be.true;
+    });
+
+    it('should emit quad every time a quad is added to the store', function () {
+      var stream = store.match();
+      var data = [];
+      stream.on('data', (quad) => {
+        data.push(quad);
+      });
+      store.addQuad('null', 'null', 'null', 'null');
+      data.should.deep.equal([new Quad('null', 'null', 'null', 'null')]);
+    });
+
+    it('should emit quad every time a matching quad is added to the store', function () {
```

Comment:
That's not how I understand the `Source` interface. Is it stated somewhere that the stream should keep on updating indefinitely?

Reply:
I guess you can interpret it like that. In RDFLib you can listen to additions and removals of the store. If that's not the intended API I can change it.